build.sbt

```diff
@@ -966,7 +966,7 @@ lazy val pkg = (project in file("lib/scala/pkg"))
   .settings(
     frgaalJavaCompilerSetting,
     scalaModuleDependencySetting,
-    compileOrder := CompileOrder.ScalaThenJava,
+    mixedJavaScalaProjectSetting,
```

Comment:
I remember we had some issues with this setting (maybe with Java modules?). Ping @Akirathan

Reply:
I've added the `PackageUtils.java` class and the build stopped compiling. Looking for inspiration in other projects that mix Java and Scala, I realized they use this `mixedJavaScalaProjectSetting`. Since then the compilation succeeds.

Reply:
Right. It is possible that the `CompileOrder.ScalaThenJava` is just a remnant. If the compilation succeeds, then everything is OK.